packages/devtools-reps/src/reps/stubs/long-string.js

```diff
@@ -9,14 +9,19 @@ const fullTextLength = multilineFullText.length;
 const initialText = multilineFullText.substring(0, 10000);
 
 const stubs = new Map();
-stubs.set("testMultiline", {
+
+const multilineLongString = {
   type: "longString",
   initial: initialText,
   length: fullTextLength,
   actor: "server1.conn1.child1/longString58"
-});
+};
+
+stubs.set("testMultiline", multilineLongString);
```

Comment:
maybe we could drop this and always use testUnloadedFullText ? What do you think ?

Reply:
In some other contexts in the test base we care about it being multiline. Hmm, probably the proper way to go is to make 2 separate stubs: one for multiline strings and one with no fullText.